templates/saas/model1/src/app/root/components/Navbar.tsx

```diff
@@ -33,10 +40,39 @@ export default function Navbar() {
 						href='/contact'
 						isActive={isActive("/contact")}
 					/>
+
+				</nav>
+				{/* Mobile Nabar */}
+
+				<nav className={`absolute !-z-40 left-0 ${displayMobileNav ? 'top-[100%]' : '-top-[500%] '} flex flex-col w-full md:flex-row justify-between py-7  px-10 md:px-6  md:pt-16  bg-white shadow-md  transition-all duration-500 ease-in-out `}>
+					<div className="flex flex-col md:flex-row gap-3">
+						<NavItem text='Home' href='/' isActive={isActive("/")} />
+						<NavItem
+							text='Pricing'
+							href='/pricing'
+							isActive={isActive("/pricing")}
+						/>
+						<NavItem
+							text='About us'
+							href='/about-us'
+							isActive={isActive("/about-us")}
+						/>
+						<NavItem
+							text='Contact'
+							href='/contact'
+							isActive={isActive("/contact")}
+						/> </div>
+					<div className="flex flex-col md:flex-row items-baseline gap-2 mt-5 md:mt-0 md:">
+						<Button text='Log in' className='bg-transparent text-black   ' />
```

Comment:
Add **"border-[1px] border-gray-500 w-[6rem]"** tailwind classes to this button please.